lib/ui/myplot/my_plot_current_stage_screen.dart

```diff
@@ -97,7 +97,7 @@ Widget buildListOfRelatedArticles(ArticleEntity articleData, MyPlotViewModel vie
 
 _buildListItemImage(ArticleEntity articleData) {
   return NetworkImageFromFuture(
-      articleData.imageUrl,
+      null,
```

Comment:
Mistake?

Reply:
I had to remove that to be able to compile, as I removed all the imageURL mess from the Article entities, now uses the imageProvider interface, I will restore this in the MyPlot work. I´ll add a TODO: